packages/fast-components-styles-msft/src/utilities/typography.ts

```diff
@@ -38,119 +38,70 @@ export interface TypeRamp {
  */
 export const typeRamp: TypeRamp = {
     t1: {
-        vp1: {
-            fontSize: 46,
-            lineHeight: 60,
-        },
-        vp3: {
-            fontSize: 62,
-            lineHeight: 72,
-        },
+        fontSize: 60,
+        lineHeight: 72,
     },
     t2: {
-        vp1: {
-            fontSize: 34,
-            lineHeight: 48,
-        },
-        vp3: {
-            fontSize: 46,
-            lineHeight: 56,
-        },
+        fontSize: 46,
+        lineHeight: 56,
     },
     t3: {
-        vp1: {
-            fontSize: 26,
-            lineHeight: 40,
-        },
-        vp3: {
-            fontSize: 34,
-            lineHeight: 48,
-        },
+        fontSize: 34,
+        lineHeight: 44,
     },
     t4: {
-        vp1: {
-            fontSize: 20,
-            lineHeight: 32,
-        },
-        vp3: {
-            fontSize: 24,
-            lineHeight: 36,
-        },
+        fontSize: 24,
+        lineHeight: 32,
     },
     t5: {
-        vp1: {
-            fontSize: 18,
-            lineHeight: 28,
-        },
-        vp3: {
-            fontSize: 20,
-            lineHeight: 28,
-        },
+        fontSize: 20,
+        lineHeight: 28,
     },
     t6: {
-        vp1: {
-            fontSize: 16,
-            lineHeight: 28,
-        },
-        vp3: {
-            fontSize: 18,
-            lineHeight: 28,
-        },
+        fontSize: 16,
+        lineHeight: 24,
     },
     t7: {
-        vp1: {
-            fontSize: 15,
-            lineHeight: 24,
-        },
+        fontSize: 14,
+        lineHeight: 20,
     },
     t8: {
-        vp1: {
-            fontSize: 12,
-            lineHeight: 20,
-        },
-        vp3: {
-            fontSize: 13,
-            lineHeight: 20,
-        },
+        fontSize: 12,
+        lineHeight: 16,
     },
     t9: {
-        vp1: {
-            fontSize: 10,
-            lineHeight: 20,
-        },
-        vp3: {
-            fontSize: 11,
-            lineHeight: 20,
-        },
+        fontSize: 10,
+        lineHeight: 12,
     },
 };
 
 /**
  * Applies a type ramp config instance based on viewport
+ * @deprecated
```

Comment:
maybe add a note that only the second argument is deprecated, not the whole function.

Reply:
@nicholasrice I intentionally deprecated this function entirely. Now that the type ramp is flat, the return value of this function matches applyTypeRampConfig. It makes the most sense to me to only keep one of the functions.